stepprogress/src/main/java/com/funrisestudio/stepprogress/StepProgressView.kt

```diff
@@ -507,40 +505,46 @@ class StepProgressView : ViewGroup, View.OnClickListener {
         return stepProgress.isAllDone()
     }
 
-    fun setTextTitlePadding(@DimenRes size: Int) {
-        textTitlePadding = resources.getDimension(size)
+    fun setTextTitlePadding(size: Float) {
+        textTitlePadding = SViewUtils.toPx(size, context)
```

Comment:
The size passed to the method should be measured in pixels so there is no need to call SViewUtils.toPx(). Same for methods below

Reply:
Fixed